.github/ISSUE_TEMPLATE.md

```diff
@@ -1,31 +1,38 @@
 ### Are you requesting a feature or reporting a bug?
+<!-- For questions, we strongly recommend StackOverflow for community support. -->
```

Comment:
> For questions...for community support

Not good - these phrases sound like they contradict each other.

```
<!-- In case you want to ask a question, we strongly recommend StackOverflow to get community support. -->
```

Reply:
I'm now sure how they contradict each other. For me, they sound similar. Would you please clarify what you mean?